### ISSUE TITLE
Train DreamBooth in SD2.1 Problem

### ISSUE THREAD

**Issue by RCBelmont** [bmaltais/kohya_ss#562]
Train DreamBooth in SD2.1 with Lion and The model will not work properly

Below is my param

accelerate launch --num_cpu_threads_per_process=2 "train_db.py" --v2 --v_parameterization --enable_bucket --pretrained_model_name_or_path="D:/Develop/AI_Dev/Webui/stable-diffusion-webui/models/Stable-diffusion/fkingScifiV2_v21f.safetensors" --train_data_dir="D:/Develop/AI_Dev/kohya_ss_GUI/UserData/DataSet/KristenStewart/TrainData" --resolution=768,768 --output_dir="D:/Develop/AI_Dev/Webui/stable-diffusion-webui/models/Stable-diffusion" --logging_dir="D:/Develop/AI_Dev/kohya_ss_GUI/UserData/DataSet/KristenStewart/Log" --save_model_as=safetensors --output_name="KristenStewart_s1252" --max_token_length=150 --max_data_loader_n_workers="0" --learning_rate="1e-7" --lr_scheduler="cosine" --train_batch_size="1" --max_train_steps="1150" --save_every_n_epochs="5" --mixed_precision="fp16" --save_precision="fp16" --caption_extension=".txt" --cache_latents --optimizer_type="Lion" --max_data_loader_n_workers="0" --max_token_length=150 --keep_tokens="1" --bucket_reso_steps=64 --xformers --bucket_no_upscale

 batch_size: 1
  resolution: (768, 768)
  enable_bucket: True
  min_bucket_reso: 256
  max_bucket_reso: 1024
  bucket_reso_steps: 64
  bucket_no_upscale: True

  [Subset 0 of Dataset 0]
    image_dir: "**********"
    image_count: 23
    num_repeats: 10
    shuffle_caption: False
    keep_tokens: 1
    caption_dropout_rate: 0.0
    caption_dropout_every_n_epoches: 0
    caption_tag_dropout_rate: 0.0
    color_aug: False
    flip_aug: False
    face_crop_aug_range: None
    random_crop: False
    token_warmup_min: 1,
    token_warmup_step: 0,
    is_reg: False
    class_tokens: KristenStewartElder
    caption_extension: .txt
